common/speech/lasr_speech_recognition_whisper/nodes/transcribe_microphone_server

```diff
@@ -110,6 +113,7 @@ class TranscribeSpeechAction(object):
             self._action_server.start()
 
     def _timer_cb(self, _) -> None:
+        return
```

Comment:
Did we decide to keep it like this permanently? If so, let's just get rid of the timer altogether.